demisto_sdk/commands/format/tests/test_formatting_yml_test.py

```diff
@@ -598,6 +598,44 @@ def test_update_docker_format(tmpdir, mocker):
     assert data['script']['dockerimage'].endswith(f':{test_tag}')
 
 
+def test_update_docker_format_with_invalid_dockerimage(requests_mock, mocker, tmp_path):
+    """Test that script and integration formatter update docker image tag
```

Comment:
please update test docstring according to `Given-When-Then` template

Reply:
done